providers/google.go

```diff
@@ -187,10 +187,8 @@ func userInGroup(service *admin.Service, groups []string, email string) bool {
 	user, err := fetchUser(service, email)
 	if err != nil {
 		logger.Printf("error fetching user: %v", err)
```

Comment:
What would this error print in the case the user is external to the organisation?

Reply:
That's a fair complaint. It'll say `Error 404: Resource Not Found: userKey, notFound`

I feel like it's still worth logging the message in case . Perhaps just prepend "Warning: " on the message? That seems to be the existing logging pattern I see.